### PR TITLE
Drafting Capabilities Text

### PR DESCRIPTION
**Pull request by nchan0154**
Continue to draft capabilities text for  Vector Features and Overlays and Interpreting Locations and Map Positions sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2232,12 +2233,22 @@ <h5>Conclusion</h5>
 <section id="capability-single-custom-image" data-ucr-role="capability">
 <h4>Display a map using a single custom image as a map layer</h4>
 <p>
-  While most web maps often are often composed of tilesets, they may also consist of one of more layers that are just single images rendered on the fly (such as from a WMS), large images that a user pans across or vector images. These kinds of maps can be useful in cases where a broader spatial context is not as important or when the author is trying to make a specific point about a set of data. Custom image layers can also be used for representing complex data for a specific area that may not be available as a complete tileset, such terrain maps or foliage coverage images produced by a specific researcher.
+While most web maps often are often composed of tilesets, they may also
```
Comment (Malvoz) [inline index.html]: "often are often", is the word often used too often here? :)

Comment (nchan0154) [reply index.html]: Thank you @Malvoz! Fixed :)

**Comment by AmeliaBR** [comment]
Hi Nic,

Can you check your editor settings to avoid inserting arbitrary line breaks?
Line breaks in paragraphs are good, but they should be [based on semantic text structure](https://sembr.org/). Otherwise, we end up with very messy diffs.

I'm going to push some clean-up edits, and then squash this PR to merge. You may need to force-pull the resulting changes onto your master again.

**Comment by AmeliaBR** [comment]
Another general comment: for the intro descriptions for _capabilities_, try to focus on _what_ it is, technologically. Don't repeat the use cases.

**Comment by AmeliaBR** [comment]
Ok, squash-merging the edits with my line break fixes.

I'm working on some content edits to the capability introductions, moving the use case descriptions over to the use case sections. (Which is good, some of those were empty!) I'll let you know when those get pushed live.

**Comment by nchan0154** [comment]
Noted on the line breaks, I had assumed it was a line length thing but this makes sense!

**Pull request merged**